Tests/kaas/k8s-default-storage-class/k8s-default-storage-class-check.py

```diff
@@ -73,23 +81,15 @@ def check_default_storageclass(k8s_client_storage):
     return default_storage_class
 
 
-def check_default_persistentvolumeclaim_readwriteonce(k8s_api_instance, storage_class):
+def create_pvc_pod(k8s_api_instance, storage_class):
     """
     1. Create PersistantVolumeClaim
     2. Create pod which uses the PersitantVolumeClaim
-    3. Check if PV got succesfully created using ReadWriteOnce
-    4. Delete resources used for testing
     """
-
-    namespace = "default"
-    pvc_name = "test-pvc"
-    pv_name = "test-pv"
-    pod_name = "test-pod"
```

Comment:
please turn these into arguments with default values, i.e.,

`namespace=NAMESPACE, pvc_name=PVC_NAME, ...`

this makes the function more versatile, and we don't litter the code with these allcaps identifiers